install_config/upgrading/migrating_etcd.adoc

```diff
@@ -84,6 +88,14 @@ and enable the 3.7 channel on the host you are running the migration from:
 # yum clean all
 ----
 
+. The migration can only be performed using *openshift-ansible* version
+3.6.173.0.21 or later. Ensure you have the latest version of the
+*openshift-ansible* packages installed:
++
+----
+# yum upgrade openshift-ansible\*
```

Comment:
Previously the 3.7 doc didn't show this step (the 3.6 doc does), so I've added in, including showing the minimum version required of openshift-ansible.